repos/tap/components/list/treeList/treeList.js

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback } from 'react'
+import { pickKeys, noPropArr } from '@keg-hub/jsutils'
+import { useTheme } from '@keg-hub/re-theme'
+import { useSelector, shallowEqual } from 'react-redux'
+import { setFeatureActive } from 'SVActions/features'
+import { loadFeature } from 'SVActions/features/loadFeature'
+import {
+  View,
+  Loading,
+  Text,
+} from 'SVComponents'
+import { Values } from 'SVConstants'
+import TreeView from 'react-native-final-tree-view'
+import { ChevronDown } from 'SVAssets/icons'
+import { useStoreItems } from 'SVHooks/store/useStoreItems'
+
+const { CATEGORIES } = Values
+
+/**
+ * Finds the node based on given id
+ * @param {string} id 
+ * @param {Array<object>} nodes 
+ * 
+ * @returns {object} node
+ */
+const findNode = (id, nodes) => {
+  let foundNode = {}
```

Comment:
Instead of creating an empty object here, then setting it in the loop below. Why not use the `__` variable instead, just rename it to `foundNode`.
That way if creates less objects, and uses `reduce` the way it's intended

Reply:
idk why i did `reduce` here. I think i was just trying stuff and forgot about it.
i changed it to use `find` instead